docs/UserGuide.adoc

```diff
@@ -73,26 +85,33 @@ Format: `list`
 === Editing a loan : `edit`
 
 In case a wrong entry is keyed in, or the customer changes their mind, this command allows details of a loan entry to be edited. +
-Format: `edit i/LOAN_ID [p/PERSON_NAME] [hp/PERSON_PHONE_NO] [e/PERSON_EMAIL] [r/RATE] [o/OBJECT_IDENTITY]`
+Format: `edit i/LOAN_INDEX [p/PERSON_NAME] [hp/PERSON_PHONE_NO] [e/PERSON_EMAIL] [r/RATE] [o/OBJECT_IDENTITY]`
 
 ****
-* Edits the loan at the specified `LOAN_ID`. This LOAN_ID refers to the LOAN_ID number shown in the displayed loan list. The LOAN_ID *must be a positive integer* 1, 2, 3, ...
+* Edits the loan at the specified `LOAN_INDEX`. This LOAN_INDEX refers to the LOAN_INDEX number shown in the displayed loan list. The LOAN_INDEX *must be a positive integer* 1, 2, 3, ...
 * At least one of the optional fields must be provided.
 * Existing values will be updated to the input values.
 * When editing tags, the existing tags of the loan will be removed i.e adding of tags is not cumulative.
 ****
 
 Examples:
 
-* `edit i/11037 o/Bicycle-017` +
-For the loan with Loan ID 11037, changes the loaned object from whatever it was before to Bicycle-017.
-* `edit i/90210 p/James Tan hp/86661352 e/alabaaoe@xyz.com r/144` +
-For the loan with Loan ID 90210, changes the customer to James Tan, who has the specified contact details, to the rate of $144 per hour.
+* `edit i/11 o/Bicycle-017` +
+For the loan at index 11, changes the loaned object from whatever it was before to Bicycle-017.
+* `edit i/9 p/James Tan hp/86661352 e/alabaaoe@xyz.com r/144` +
+For the loan at index 9, changes the customer to James Tan, who has the specified contact details, to the rate of $144 per hour.
+
+[NOTE]
+====
+We do not allow editing of the loan's start time or the return time because we want you to be able to use our application as an auditing tool. We hope that this would simplify your business. However, by allowing you to edit the start and end time, this would invalidate this application as an auditing tool.
+
+We are currently looking into alternatives that would allow you to edit the start and end time though! Stay tuned for our announcement in LoanBook version 2.0!
```

Comment:
Sounds like it contradicts your stance in the previous paragraph.

Reply:
From what I was thinking, there could be other alternatives where this is available. Perhaps it may be better to not mention this as a potential future feature.